src/int.rs

```diff
@@ -0,0 +1,266 @@
+//! This module contains an 256-bit signed integer implementation.
+
+use crate::errors::ParseI256Error;
+use std::fmt;
+use std::str::{self, FromStr};
+use web3::types::U256;
+
+/// Compute the two's complement of a U256.
+fn twos_complement(u: U256) -> U256 {
+    let (twos_complement, _) = (u ^ U256::max_value()).overflowing_add(U256::one());
```

Comment:
I believe the purpose of `u ^ U256::max_value()` is to negate each bit. `!u` would be clearer. Should exist according to `https://docs.rs/primitive-types/0.7.0/primitive_types/struct.U256.html#impl-Not`.